utils/tracing/go/tracing_test.go

```diff
@@ -2,32 +2,61 @@ package tracing
 
 import (
 	"context"
+	"fmt"
 	"log"
 	"os"
+	"os/exec"
 	"testing"
 	"time"
 
 	"go.opentelemetry.io/otel"
 	"go.opentelemetry.io/otel/attribute"
 )
 
+func TestMain(m *testing.M) {
+	cmd := exec.Command("docker", "pull", "openzipkin/zipkin")
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
```

Comment:
use CombinedOutput API instead

Reply:
I dont think CombinedOutput can be used to set the output of the cmd, it can only capture cmd output. How specifically do you want me to use CombinedOutput?